udata/core/reuse/models.py

```diff
@@ -23,6 +25,20 @@ class ReuseQuerySet(OwnedQuerySet):
     def visible(self):
         return self(private__ne=True, datasets__0__exists=True, deleted=None)
 
+    def visible_by_user(self, user: User) -> OwnedQuerySet:
+        """Return EVERYTHING visible to the user."""
+        public_qs: OwnedQuerySet = db.Q(private__ne=True, deleted=None, archived=None)
+        if user.is_anonymous:
+            return self(public_qs)
+
+        owners: list[User | Organization] = []
+        owned_qs: OwnedQuerySet = db.Q()
+        owners = list(user.organizations) + [user.id]
+        for owner in owners:
+            owned_qs |= db.Q(owner=owner) | db.Q(organization=owner)
```

Comment:
Can the DB use `user` instead of `owner`, so that the logic could be like `owned_qs |= db.Q(user=owner) | db.Q(organization=owner)`?

Reply:
I'm not sure I understand your point? `owner` is a field on the `Reuse` model, but `user` is not, what are you suggesting?